client/python/lib/ovmsclient/tfs_compat/grpc/serving_client.py

```diff
@@ -105,7 +116,111 @@ def get_model_status(self, request):
 
     @classmethod
     def _build(cls, config):
-        raise NotImplementedError
+        
+        _check_config(config)
+
+        grpc_address = f"{config['address']}:{config['port']}"
+
+        if 'tls_config' in config:
+            client_key, client_cert, server_cert = _prepare_certs(
```

Comment:
The call with parameters "to be evaluated" is hard to read. 
Could you take values extraction out and just call `_prepare_certs(server_cert_path, client_key_path, client_cert_path) `?
See: https://www.tutorialspoint.com/python/dictionary_get.htm to provide default `None` if key has not been found in the dict.

Also, the required parameters should go first and possible `Nones` should be placed at the end.